packages/server-reason-react-ppx/cram/component-definition.t/input.re

```diff
@@ -68,16 +68,25 @@ module Upper_with_aria = {
   let make = (~children) => <div ariaHidden="true"> children </div>;
 };
 
-module Form_with_method = {
+// As the ppx does not runs as preprocessor, using method will break, like the error below:
+// +  File "output.ml", line 62, characters 33-41:
+// +  62 |     let make ~children  = ((form ~method:"GET" ~children:[children] ())
```

Comment:
`~method` is invalid in OCaml, since needs to be `~method_`. In Reason, this isn't in the best case where method is valid, when often OCaml keywords are invalid in Reason